src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.AutoRedirect.cs

```diff
@@ -482,6 +538,12 @@ public async Task HttpClientHandler_CredentialIsNotCredentialCacheAfterRedirect_
         [Theory, MemberData(nameof(RemoteServersAndRedirectStatusCodes))]
         public async Task GetAsync_CredentialIsCredentialCacheUriRedirect_StatusCodeOK(Configuration.Http.RemoteServer remoteServer, int statusCode)
         {
+            if (statusCode == 308 && (IsWinHttpHandler && PlatformDetection.WindowsVersion < 10))
```

Comment:
Is there a more specific version of Windows 10 that we should be testing for?

Reply:
Added in [this commit](https://github.com/dotnet/runtime/commit/42e3d1132110b0bef0c3c9758a77f605c066c357).
I don't know why there is ```< 10```, nor does the original [issue](https://github.com/dotnet/corefx/issues/30389) helps me understand it.
Do you want me to investigate it further?

Reply:
Considering it's only for tests, it's probably not worth the effort. Thanks for taking a look.